src/ResourceManager/GuestConfiguration/Commands.GuestConfiguration/ChangeLog.md

```diff
@@ -17,5 +17,9 @@
     * Overview of change #1
         - Additional information about change #1
 -->
-## Current Release
- - First release of Get-AzureRmGuestConfigurationPolicy report cmdlet (version: 0.9.0-preview)
+## Current release 
+ - Return only the latest and detailed reports for all policies assigned to VM (optionally, by initiative id or name) by default with Get-AzureRmVmGuestPolicyReport cmdlet
+ - Remove Detailed and Latest switches of Get-AzureRmVmGuestPolicyReport cmdlet
+
+## 0.9.0-preview : September 2018
```

Comment:
Please only add to the changelog, do not add additional headers.

Reply:
changed. could you check ? Is this what needed